src/pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -459,8 +459,15 @@ def _update_learning_rates(
             if config.interval == interval and current_idx % config.frequency == 0:
                 monitor_val = None
                 if config.reduce_on_plateau:
-                    # If instance of ReduceLROnPlateau, we need a monitor
                     monitor_key = config.monitor
+                    if monitor_key is None:
+                        raise RuntimeError(
+                            "If you use `ReduceLROnPlateau` for updating a learning rate,"
+                            " you must specify a monitor. Please see https://pytorch-lightning.readthedocs.io"
+                            "/en/stable/api/pytorch_lightning.core.LightningModule.html"
+                            "#pytorch_lightning.core.LightningModule.configure_optimizers as well."
+                        )
+
                     monitor_val = self._get_monitor_value(monitor_key)
```

Comment:
can you change the return type of _get_monitor_value to `Optional[Any]` too?

Reply:
Thank you for pointing out, changed in 4b947ac. In my understanding, We still need to `Any` since `Trainer.callback_metrics` returns `dict` (key and value are untyped). Is this correct?

Reply:
Yes, Optional[Any]